packages/plugins/actions/actions-core/src/actions/testContext.js

```diff
@@ -0,0 +1,71 @@
+/*
+  Copyright 2020-2021 Lowdefy, Inc
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+
+import { WebParser } from '@lowdefy/operators';
```

Comment:
I think it would be best if we can import `createContext` from `@lowdefy/engine`, and create a "real" Lowdefy context, otherwise we need to duplicate complicated implementation logic in engine here that can easily get out of sync.